locations/spiders/tgjones_gb.py

```
@@ -25,7 +25,7 @@ class TgjonesGBSpider(SitemapSpider):
     }
     coordinates_pattern = re.compile(r"google\.maps\.LatLng\(\s*([-\d.]+)[,\s]+([-\d.]+)\s*\)")
     skip_auto_cc_domain = True
-    requires_proxy = True
+    #requires_proxy = True
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)